### PR TITLE
!requiretty

### PR DESCRIPTION
**Pull request by manasdk**

### CODE REVIEW

**Comment by DoriftoShoes** [comment]
+1

File: tools/st2_deploy.sh [hunk 1]
```diff
@@ -147,6 +147,9 @@ create_user() {
     then
       echo "${SYSTEMUSER}    ALL=(ALL)       NOPASSWD: ALL" >> /etc/sudoers.d/st2
     fi
+    # make sure requiretty is disabled.
+    # unable to get ^Defaults[\s]+requiretty to work.
+    sed -i "s/^Defaults    requiretty/# Defaults    requiretty/g" /etc/sudoers
```
Comment (lakshmi-kannan) [inline tools/st2_deploy.sh]: Ghetto!

Comment (Kami) [inline tools/st2_deploy.sh]: Probably safer to use a regex for whitespace (\s+) in case there are more or less whitespace characters.

Comment (manasdk) [inline tools/st2_deploy.sh]: I tried 

`^Defaults[\s]+requiretty`, `^Defaults(\s)+requiretty` and `^Defaults(\s+)requiretty`

Neither worked so I gave up.

Comment (Kami) [inline tools/st2_deploy.sh]: `sed -i "s/^Defaults\s\+requiretty/# Defaults    requiretty/g" /etc/sudoers`

Should do it I believe.

Comment (Kami) [inline tools/st2_deploy.sh]: I tested the one I posted above locally with GNU sed and it works fine.

**Pull request merged**